src/routingmanager/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Does this file need to be checked in or can it be kept locally? Is it needed for UTs to run?

Reply:
It is not needed.  Visual Studio added for some reason and I missed it.

Sorry been very busy, is it possible someone else can update the PR?